doc/developers/utilities.rst

```diff
@@ -144,7 +144,7 @@ efficiently process ``scipy.sparse`` data.
 
 - :func:`sparsefuncs.inplace_csr_row_normalize_l1` and
   :func:`sparsefuncs.inplace_csr_row_normalize_l2`: can be used to normalize
-  individual sparse samples to unit l1 or l2 norm as done in
+  individual sparse samples to unit L1 or L2 norm as done in
```

Comment:
It's up to you. I noticed L2 was used earlier in the doc, so I wanted to make the usage consistent. The uppercase L has always struck me as a bit easier to read.